Gifski/Constants.swift

```diff
@@ -6,4 +6,5 @@ extension NSColor {
 
 extension Defaults.Keys {
 	static let outputQuality = Defaults.Key<Double>("outputQuality", default: 1)
+	static let totalConversionsCount = Defaults.Key<Int>("totalConversionsCount", default: 0)
```

Comment:
I prefer `successfulConversionsCount` or just `conversionsCount`. I think the key itself implies that it’s the total amount.